CHANGELOG.md

```diff
@@ -1,3 +1,13 @@
+<a name="1.0.3"></a>
+## [1.0.3](https://github.com/daisy/ace/compare/v1.0.2...v1.0.3) (2019-05-01)
```

Comment:
The changelog is generated at release time using [conventional-changelog](https://github.com/conventional-changelog/conventional-changelog), which is based on our following of [Angular's commit message guidelines](https://github.com/angular/angular/blob/master/CONTRIBUTING.md#commit)… so no need to update the changelog manually here :-)

Reply:
I am going to revert this change, and I think that when the PR gets merged, we can rebase and create a suitable commit message for this PR?

Reply:
Done: https://github.com/daisy/ace/pull/226/commits/c54ef03204c9564c8ea9890e696db6f8ba9de7c5